src/functionalTest/java/org/jfrog/gradle/plugin/artifactory/tests/PluginAndroidTest.java

```diff
@@ -0,0 +1,74 @@
+package org.jfrog.gradle.plugin.artifactory.tests;
+
+import org.gradle.testkit.runner.BuildResult;
+import org.jfrog.build.api.dependency.PropertySearchResult;
+import org.jfrog.build.extractor.ci.BuildInfo;
+import org.jfrog.build.extractor.ci.Module;
+import org.jfrog.gradle.plugin.artifactory.GradleFunctionalTestBase;
+import org.jfrog.gradle.plugin.artifactory.TestConstant;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.gradle.testkit.runner.TaskOutcome.FAILED;
+import static org.jfrog.gradle.plugin.artifactory.TestConstant.EXPECTED_ANDROID_ARTIFACTS;
+import static org.jfrog.gradle.plugin.artifactory.TestConstant.GRADLE_ANDROID_VERSION;
+import static org.jfrog.gradle.plugin.artifactory.utils.ValidationUtils.getBuildInfo;
+import static org.testng.Assert.*;
+
+public class PluginAndroidTest extends GradleFunctionalTestBase {
+    @Test
+    public void androidTest() throws IOException {
+        runPublishTest(GRADLE_ANDROID_VERSION, TestConstant.ANDROID_GRADLE_EXAMPLE, this::checkBuildResults);
```

Comment:
I can't see this new reource project using the plugin, should we apply the CI test to it?
In addition, I would add another test for CI as well if we already have a new andorid project as a resource